Gopkg.lock

```diff
@@ -2,21 +2,21 @@
 
 
 [[projects]]
-  digest = "1:5528a950c822b95fc7108dd7ad2f6e9e0127e0f104808e81ec812953447c7eee"
+  digest = "1:72c4b04a1acbfcf1c294b1301f9cca730ebcdcc3e719e9927ae57a1393bf9cc0"
```

Comment:
just noticed that we don't list this dependency in gopkg.toml package as the other direct dependencies.

Reply:
Yeah it happened a while ago when I added azure. Go dep doesn't treat these imports:
```
	"github.com/Azure/azure-sdk-for-go/profiles/latest/compute/mgmt/compute"
	"github.com/Azure/azure-sdk-for-go/profiles/latest/network/mgmt/network"
```

as  normal imports, so it removes the `azure-sdk-for-go` package all the time you `dep ensure`, however, it is used and that's why is in the .lock. Not sure if this is because of the way the azure go client is made in modules.